<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create</class>
 <widget class="QWidget" name="create">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>signup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
background:rgba(6, 16, 28, 0.8);
border-radius:15px;
}</string>
  </property>
  <widget class="QPushButton" name="back2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QFrame" name="bgframe">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;
background:setAttribute(Qt::WA_TranslucentBackground)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="signupL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>87</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c2c2c2;&quot;&gt;Signup&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>99</y>
      <width>151</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
    </property>
    <widget class="QLineEdit" name="usernameRF">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>Enter Your Name </string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailRF">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>37</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter Your  Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwRF">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>65</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter Your Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwcRF">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>93</y>
       <width>133</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password Confirm </string>
     </property>
    </widget>
    <widget class="QComboBox" name="profboxR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>121</y>
       <width>98</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Volunteer</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Volunteer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Consumer</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>99</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
    </property>
    <widget class="QLabel" name="unR">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c2c2c2;&quot;&gt;User Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="emailR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c2c2c2;&quot;&gt;Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="pwR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c2c2c2;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="pwcR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c2c2c2;&quot;&gt;Password Confirm &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="profR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c2c2c2;&quot;&gt;Profile&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="signupR">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color:white;
border-radius:10px;
}
QPushButton:hover
{
color:#333;
border-radius:10px;
background:#49ebff;
}
QPushButton

{
background:skyblue;
}</string>
    </property>
    <property name="text">
     <string>Sign-up</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
